### PR TITLE
card component created

### PR DESCRIPTION
**Pull request by psubham**

### CODE REVIEW

File: Lpicks-Core/src/app/core/card/card.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  styleUrls: ['card.scss'],
+  host: {
+    'class': 'kdi-card'
+  }
+})
+export class KdiCard {
+
+}
+
+@Directive({
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/core/card/card.ts]: Do we need all these directives

File: Lpicks-Core/src/app/demo/DemoCardModule.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
+import { KdiCard, KdiCardFooter} from '../core/card/card';
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/demo/DemoCardModule.ts]: Correct the indentation

File: Lpicks-Core/src/app/demo/DemoCardModule.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        DemoCardComponent,
+        KdiCard,
+        KdiCardFooter,
+      
+        KdiCardContent,
+        KdiCardTitle
+    ],
+    imports: [],
+    providers: [],
+    bootstrap: [DemoCardComponent]
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/demo/DemoCardModule.ts]: Why are we bootsrapping this?

File: Lpicks-Core/src/app/demo/DemoCardModule.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import { KdiCard, KdiCardFooter} from '../core/card/card';
+
+import { DemoCardComponent } from './demo-card.component';
+import {  KdiCardContent, KdiCardTitle } from '../core/card/card';
+@NgModule({
+    declarations: [
+        DemoCardComponent,
+        KdiCard,
+        KdiCardFooter,
+
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/demo/DemoCardModule.ts]: This needs . to be cleaned up, fix indentation etc.

File: Lpicks-Core/src/app/demo/demo-card.component.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-card',
+  templateUrl: './demo-card.component.html',
+ 
+})
+export class DemoCardComponent {
+ // title = 'Lpicks-Core';
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/demo/demo-card.component.ts]: Remove commented code

File: Lpicks-Core/src/app/app.module.ts [hunk 6]
```diff
@@ -3,14 +3,17 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { DemoCardComponent } from './demo/demo-card.component';
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/app.module.ts]: Unused import

Comment (psubham) [reply Lpicks-Core/src/app/app.module.ts]: i have removed the unused imports

File: Lpicks-Core/src/app/core/card/card.ts (hunk 1 above)
Comment (psubham) [reply Lpicks-Core/src/app/core/card/card.ts]: yes, to differentiate card header,body and footer. As I am not using  any Java Script Object.

File: Lpicks-Core/src/app/demo/DemoCardModule.ts (hunk 2 above)
Comment (psubham) [reply Lpicks-Core/src/app/demo/DemoCardModule.ts]: corrected the indentation.

File: Lpicks-Core/src/app/demo/DemoCardModule.ts (hunk 3 above)
Comment (psubham) [reply Lpicks-Core/src/app/demo/DemoCardModule.ts]: its not needed so,i removed it,

File: Lpicks-Core/src/app/demo/demo-card.component.ts (hunk 5 above)
Comment (psubham) [reply Lpicks-Core/src/app/demo/demo-card.component.ts]: removed the commented code

File: Lpicks-Core/src/app/demo/demo-card-demo/demo-card.component.scss [hunk 7]
```diff
@@ -0,0 +1,15 @@
+.temp{
+    width:auto; 
+    padding: 10px;
+    margin:10px;
+}
+.container{  
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    /* margin: auto; */
```
[the remark this answers is not in the document]
Comment (psubham) [reply Lpicks-Core/src/app/demo/demo-card-demo/demo-card.component.scss]: removed the comment

File: Lpicks-Core/src/app/demo/demo.module.ts [hunk 8]
```diff
@@ -1,8 +1,10 @@
 import { NgModule } from '@angular/core';
 import { DropdownDemoComponent } from './dropdown-demo/dropdown-demo.component';
 import { CoreModule } from '../core/core.module';
+import { DemoCardComponent } from './demo-card-demo/demo-card.component';
+import { PageComponent } from './page/page.component';
 
-const COMPONENTS = [DropdownDemoComponent];
+const COMPONENTS = [DropdownDemoComponent,DemoCardComponent,PageComponent];
```
[the remark this answers is not in the document]
Comment (psubham) [reply Lpicks-Core/src/app/demo/demo.module.ts]: fix the spaces

File: Lpicks-Core/src/app/demo/page/page.component.html [hunk 9]
```diff
@@ -0,0 +1,12 @@
+<div class="main-editor">
+    <button class="btn">
+      run
+    </button>
+    <div class="first" contenteditable="true" [textContent]="model" (input)="model=$event.target.textContent">
+        
+    </div>
+    <iframe class="second" frameborder="0">
```
[the remark this answers is not in the document]
Comment (psubham) [reply Lpicks-Core/src/app/demo/page/page.component.html]: removed the iframe

File: Lpicks-Core/src/app/app.component.html [hunk 10]
```diff
@@ -1,3 +1,8 @@
+<!-- <demo-card> </demo-card>
```
[the remark this answers is not in the document]
Comment (psubham) [reply Lpicks-Core/src/app/app.component.html]: for checking the live demo i comment other code

**Pull request merged**